code/__DEFINES/atmospherics.dm

```diff
@@ -15,123 +14,105 @@
+#define SPACE_SUIT_MIN_TEMP_PROTECT			2.0		//what min_cold_protection_temperature is set to for space-suit quality jumpsuits or suits. MUST NOT BE 0.
+#define SPACE_SUIT_MAX_TEMP_PROTECT			1500
+
+#define FIRE_SUIT_MIN_TEMP_PROTECT			60		//Cold protection for firesuits
+#define FIRE_SUIT_MAX_TEMP_PROTECT			30000	//what max_heat_protection_temperature is set to for firesuit quality suits. MUST NOT BE 0.
+#define FIRE_HELM_MIN_TEMP_PROTECT			60		//Cold protection for fire helmets
+#define FIRE_HELM_MAX_TEMP_PROTECT			30000	//for fire helmet quality items (red and white hardhats)
+
+#define FIRE_IMMUNITY_SUIT_MAX_TEMP_PROTECT	35000	//what max_heat_protection_temperature is set to for firesuit quality suits. MUST NOT BE 0.
+#define FIRE_IMMUNITY_HELM_MAX_TEMP_PROTECT	35000	//for fire helmet quality items (red and white hardhats)
+
+#define HELMET_MIN_TEMP_PROTECT				160		//For normal helmets
+#define HELMET_MAX_TEMP_PROTECT				600		//For normal helmets
+#define ARMOR_MIN_TEMP_PROTECT				160		//For armor
+#define ARMOR_MAX_TEMP_PROTECT				600		//For armor
+
+#define GLOVES_MIN_TEMP_PROTECT				2.0		//For some gloves (black and)
+#define GLOVES_MAX_TEMP_PROTECT				1500	//For some gloves
+#define SHOES_MIN_TEMP_PROTECT				2.0		//For gloves
+#define SHOES_MAX_TEMP_PROTECT				1500	//For gloves
+
 
-#define NORMPIPERATE 30					//pipe-insulation rate divisor
-#define HEATPIPERATE 8					//heat-exch pipe insulation
+#define PRESSURE_DAMAGE_COEFFICIENT			4		//The amount of pressure damage someone takes is equal to (pressure / HAZARD_HIGH_PRESSURE)*PRESSURE_DAMAGE_COEFFICIENT, with the maximum of MAX_PRESSURE_DAMAGE
+#define MAX_HIGH_PRESSURE_DAMAGE			4		//This used to be 20... I got this much random rage for some retarded decision by polymorph?! Polymorph now lies in a pool of blood with a katana jammed in his spleen. ~Errorage --PS: The katana did less than 20 damage to him :(
+#define LOW_PRESSURE_DAMAGE					2		//The amounb of damage someone takes when in a low pressure area (The pressure threshold is so low that it doesn't make sense to do any calculations, so it just applies this flat value).
 
-#define FLOWFRAC 0.99				// fraction of gas transfered per process
+#define COLD_SLOWDOWN_FACTOR				20		//Humans are slowed by the difference between bodytemp and BODYTEMP_COLD_DAMAGE_LIMIT divided by this
 
-#define TANK_LEAK_PRESSURE		(30.*ONE_ATMOSPHERE)	// Tank starts leaking
-#define TANK_RUPTURE_PRESSURE	(40.*ONE_ATMOSPHERE) // Tank spills all contents into atmosphere
+// Atmos pipe limits
+#define MAX_OUTPUT_PRESSURE					4500 // (kPa) What pressure pumps and powered equipment max out at.
+#define MAX_TRANSFER_RATE					200 // (L/s) Maximum speed powered equipment can work at.
 
-#define TANK_FRAGMENT_PRESSURE	(50.*ONE_ATMOSPHERE) // Boom 3x3 base explosion
-#define TANK_FRAGMENT_SCALE	    (10.*ONE_ATMOSPHERE) // +1 for each SCALE kPa aboe threshold
-								// was 2 atm
\ No newline at end of file
+// Sound Limits
+#define SOUND_MINIMUM_PRESSURE 10
```

Comment:
:+1: to all of this- I intended to clean it out at some point, it was just copypasted from setup.dm